apps/web/lib/components/button.tsx

```diff
@@ -92,12 +91,13 @@ export function RoundedButton({ children, className, ...rest }: RoundedButtonPro
 }
 
 export function BackButton({ onClick, className }: { onClick?: () => void; className?: string }) {
-	const t = useTranslations();
 
 	return (
-		<button type="button" className={clsxm('flex items-center', className)} onClick={onClick}>
+		<button type="button" className={clsxm('flex items-center justify-start text-sm', className)}>
 			<ArrowLeft className="mr-2" />
-			<span className="text-sm">{t('common.BACK')}</span>
+			<span className="text-sm" onClick={onClick}>
+				Back
```

Comment:
Here @Cedric921 you actually should use common.BACK with value "Back" etc.

Reply:
resolved